src/chart/speclayout/bubblechart_impl.h

```diff
@@ -16,7 +16,6 @@ class BubbleChartImpl : public BubbleChart
 {
 public:
 	BubbleChartImpl(const std::vector<double> &sizes,
```

Comment:
`explicit` jelölés kell, mert már 1 paraméteres is lehet

Reply:
ok